heron_description/urdf/base.xacro

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <!--
+    Vehicle's parameters (remember to enter the model parameters below)
+  -->
+
+  <xacro:property name="mass" value="28"/>
+  <!-- Center of gravity -->
+  <xacro:property name="cog" value="0 0 -0.13"/>
+  <!-- Fluid density -->
+  <xacro:property name="rho" value="1028"/>
+
+  <!-- Loading vehicle's specific macros -->
+  <xacro:include filename="$(find heron_description)/urdf/snippets.xacro"/>
+
+
+  <!--
+    Visual mesh file for the vehicle, usually in DAE (Collada) format. Be sure to store the
+    mesh with the origin of the mesh on the same position of the center of mass, otherwise
+    the mesh pose will have to be corrected below in the <visual> block.
+    Open the meshes for the RexROV vehicle in Blender to see an example on the mesh placement.
+  -->
+  <xacro:property name="visual_mesh_file" value="file://$(find heron_description)/meshes/heron_base.stl"/>
+
+  <!-- Collision geometry mesh, usually in STL format (it is recommended to keep
+  this geometry as simple as possible to improve the performance the physics engine
+  regarding the computation of collision forces) -->
+  <xacro:property name="collision_mesh_file" value="file://$(find heron_description)/meshes/heron_base.stl"/>
```

Comment:
Does the simulator use the collision mesh to calculate buoyancy?  If so, simplifying this mesh may go a long way for efficiency